app-misc/qmodmaster/qmodmaster-0.5.2.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="A free Qt-based implementation of a ModBus master application"
+DESCRIPTION="A graphical user interface allows easy communication with ModBus \
+ RTU and TCP slaves. QModMaster also includes a bus monitor for examining all  \
+ traffic in the bus
+ QModMaster is based on libmodbus. for modbus  \
+ communication and on QsLog for logging.
+ Supports Haiku, Windows and Linux.
+
+-Features
+     ModBus RTU/TCP Master
+     Monitor Bus activity [Raw Data and PDU details]
+"
+COPYRIGHT="2012-2020 Elbar"
+HOMEPAGE="https://sourceforge.net/projects/qmodmaster/"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://download.sourceforge.net/qmodmaster/qModMaster-code-0.5.2-3/qModMaster-code-$portVersion-3.zip
```

Comment:
The provided link doesn't work for me? ```https://sourceforge.net/projects/qmodmaster/files/qModMaster-code-0.5.2-3.zip"``` does work however ...

Reply:
sorry all, i have problem with my box..  .. and i make mistake .. seem i upload untested one.. i will try test this week

Reply:
fixed url.. thanks